ocs/ocs_agent.py

```diff
@@ -739,6 +747,17 @@ def encoded(self):
 
 SESSION_STATUS_CODES = [None, 'starting', 'running', 'stopping', 'done']
 
+
+class OpCode(Enum):
+    NONE = 1
```

Comment:
I started at one because the [Enum documentation](https://docs.python.org/3/library/enum.html) always starts at one, so I thought there might be something weird with enum values of zero. But if you'd prefer I can start at 0.

Reply:
Ah, I see why they do this now:

> The reason for defaulting to 1 as the starting number and not 0 is that 0 is False in a boolean sense, but enum members all evaluate to True.

I don't think it would cause us any problems, but probably best to leave it as is.